lib/build.gradle

```diff
@@ -41,7 +41,7 @@ dependencies {
     compile 'com.android.support:design:25.3.1'
     compile 'com.google.code.gson:gson:2.8.0'
     compile 'com.squareup:otto:1.3.8'
-    compile 'com.auth0.android:auth0:1.8.0'
+    compile 'com.auth0.android:auth0:1.8.0-SNAPSHOT'
```

Comment:
That's pointing to the local dependency until we release 1.9.0, then CI passes 👍